### ISSUE TITLE
Error after moving all services to unRaid.

### ISSUE THREAD

**Issue by DK2Maverick** [Notifiarr/notifiarr#133]
They were all working when they were stand alone dockers on a ubuntu box. I moved them all unRaid yesterday and noticed this error. If I comment out Radarr from the config. Only the top error goes away. Note - Notifiarr is in a docker on the ubuntu box.
```
[ERROR] 2021/11/16 18:14:15 Radarr URL fixed: http://192.168.1.241:7878/ -> http://192.168.1.241:7878 (continuing)
[ERROR] 2021/11/16 18:12:40 Go Panic! This is a bug please report it on github: https://github.com/Notifiarr/notifiarr/issues/new

runtime error: invalid memory address or nil pointer dereference

goroutine 1 [running]:
runtime/debug.Stack()
	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
main.main.func1()
	/notifiarr/main.go:21 +0x3d
panic({0x87eda0, 0xc9d800})
	/usr/local/go/src/runtime/panic.go:1038 +0x215
github.com/Notifiarr/notifiarr/pkg/apps.(*Apps).Setup(0xc000240000, 0x8cd460)
	/notifiarr/pkg/apps/apps.go:177 +0x374
github.com/Notifiarr/notifiarr/pkg/configfile.(*Config).Get(0xc0002304e0, {0x90c498, 0x16}, {0x9030c7, 0x2})
	/notifiarr/pkg/configfile/config.go:95 +0x1bb
github.com/Notifiarr/notifiarr/pkg/client.(*Client).loadConfiguration(0xc000214980)
	/notifiarr/pkg/client/start.go:137 +0x1dd
github.com/Notifiarr/notifiarr/pkg/client.(*Client).start(0xc000214980)
	/notifiarr/pkg/client/start.go:81 +0x36
github.com/Notifiarr/notifiarr/pkg/client.Start()
	/notifiarr/pkg/client/start.go:76 +0xae
main.main()
```

**Comment by davidnewhall** [comment]
Please share your config.

**Comment by DK2Maverick** [comment]
# This API key must be copied from your notifiarr.com account.
api_key = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx"

bind_addr = "0.0.0.0:5454"

auto_update = "off"

quiet = false

urlbase = "/"

upstreams = [ "127.0.0.1/32", "::1/128" ]

log_file_mb = 100

log_files = 0

file_mode = "0600"

timeout = "1m0s"

##################
# Starr Settings #
##################

[[lidarr]]
name     = "Lidarr" # Set a name to enable checks of your service.
url      = "http://192.168.1.241:8686/"
api_key  = "d2fd8affa04140e9af7e6fe8d926ad19"

[[radarr]]
name      = "Radarr" # Set a name to enable checks of your service.
url       = "http://192.168.1.241:7878/"
api_key   = "37fc2853fd074d8b8ee9326c4ecafba2"


[[readarr]]
name      = "Readarr" # Set a name to enable checks of your service.
#url       = "http://192.168.1.241:8787/readarr"
#api_key   = "3f36698abed5442088956193c43f487c"

[[sonarr]]
name      = "Sonarr"  # Set a name to enable checks of your service.
url       = "http://192.168.1.241:8989/"
api_key   = "cba4a4e368a64cb5813a504181eb6828"

[[bazarr]]
name      = "Bazarr"  # Set a name to enable checks of your service.
url       = "http://192.168.1.188:6767/"
api_key   = "47226a8889701856b977d3a483a03339"


# Download Client Configs (below) are used for dashboard state and service checks.

[[deluge]]
name     = "Deluge"  # Set a name to enable checks of your service.
url      = "http://192.168.1.241:8112/"
password = "deluge"


#[[qbit]]
#name     = ""  # Set a name to enable checks of your service.
#url      = "http://qbit:8080/"
#user     = ""
#pass     = ""


#[[sabnzbd]]
#name     = "sabnzbd"  # Set a name to enable checks of this application.
#url      = "http://192.168.1.188:8070/"
#api_key  = "553505c8dbbf4721937e2d24800dc363"

##################
# Service Checks #
##################

[services]
  disabled = false   # Setting this to true disables all service checking routines.
  parallel = 1       # How many services to check concurrently. 1 should be enough.
  interval = "10m0s" # How often to send service states to Notifiarr.com. Minimum = 5m.
  log_file = ''      # Service Check logs go to the app log by default. Change that by setting a services.log file here.


## Another example. Remember to uncomment [[service]] if you use this!
##
[[service]]
  name    = "Bazarr"
  type    = "http"
  check   = 'http://192.168.1.188:6767/series/'
  expect  = "200"
  timeout = "10s"

**Comment by austinwbest** [comment]
You left your api keys in there but as for the problem.. My guess is the upstream setting...

**Comment by DK2Maverick** [comment]
yeah saw the keys so I edited them out. Thanks  - I will experiment with the upstream.

**Comment by davidnewhall** [comment]
There is definitely a bug here. it's not `upstreams`. I think it's this:
```
[[readarr]]
name      = "Readarr" # Set a name to enable checks of your service.
#url       = "http://192.168.1.241:8787/readarr"
#api_key   = "xxxxxxxxxxx"
```

You set a `name` but not a `url`. Try removing that stanza.

**Comment by DK2Maverick** [comment]
That was it. Thanks for the help. I stared at the file for half the night and didn't see anything different from before I moved the arrs to unRaid. Completely forgot that I turned Readarr on at the same time, and did not see the comment marks. Stared at them on and off for hours and didn't notice them till you pointed it out. TBH I didn't see them  after you pointed it out, till my son walked by and said "you have comment marks on those lines".....

All works now... Thanks again
